### DISCUSSION TITLE
[Unstable] Re-implement pressure immunity for Clown power talent tree.

### DISCUSSION THREAD

**Discussion by Padex98** [FakeFishGames/Barotrauma#10231] [Unstable feedback]
It's a small suggestion, but I think it would make a big difference, in making the clown perk be as unique and quirky as it currently is in the stable release. 
While pressure immunity does sound like a powerful perk to have, it has the drawbacks of still requiring a full clown suit, having to replace oxygen, and not having the damage resistances a suit would have.
It's no different than wearing a suit all the time, but without the advantages a suit gives, except for the pressure immunity.

**Comment by MarkCringan** [comment]
I don't know, I always felt that not wearing a suit whilst not outside of the submarine was a bit immersion breaking imo. There's no real reason a clown should be able to do that.

**Reply by Padex98** [reply]
I mean, it's not that game breaking, especially now that people can only choose one talent tree, so not all players who choose assistant necessairily will get that perk

**Reply by AhpiTai** [reply]
Imo the clown having such an absurd power such as innate pressure immunity gave the clown an eldritch feel. As if its powers are unwordly like Pennywise. Matches the feel of the game. Let alone the fact we already have pressure stabilizers in-game.

**Comment by six-pd** [comment]
I second this, the clown tree is not meant to be realistic or serious. Clown gear is inferior to any dive suit, so not using it doesn't really have much benefit, but it is pretty funny to see a clown swimming around through a periscope, and I think this kind of tomfoolery is what makes Barotrauma clowns so iconic.

Clowns already break the immersion of a real submarine crew, and most serious RP games will not allow clowns either way, whether they can swim naked or not.

**Comment by Marrynd** [comment]
I made a Reddit post with this exact topic https://www.reddit.com/r/Barotrauma/comments/ytzo2i/please_bring_back_water_prankster_pressure/

I would say I really like the new clown talent, switching from being an overpower terminator to an actual clown is a step in the right direction

but taking away water prankster immunity is like taking away a part of clown personality, it is not essential to gameplay nor was it overpowered, it is just a funny prank you can pull on your crew, I understand the argument that it breaks immersion but that's precisely what it is so funny, no captain expect to see a clown alive and well through his command window, it's funny as hell.

it's the reason I decided to pick clown in my first voyage with my friend after the update drop, we got a good laugh out of that one.

please don't take it away, not for gameplay reasons, just keep it for the lore reason, a proof that honk mother existed.
"Worship of the Honkmother is a pathway to many abilities some consider to be... unnatural."

**Comment by Instant2116** [comment]
Any overpowered ability should NOT be retained. Killing Moloch in brawl fans won't ever be satisfied, after speedrunning Endworm with a one-shot toy hammer in 15 seconds. Moreover most of clown players don't even know that honk mother religion about overcome through suffering, not to be a stats god with pressure immunity, 200 (older 400) in every skill and chance to one-shot any creature.

**Reply by MapleWheels** [reply]
Getting skills to 200 is a bit much but it's a compensatory mechanism for Assistant for the fact that they don't have the other perks other classes do. With bonuses, most jobs can hit 130-170 skill in most perk regardless so it's fine. At those levels the skill level stops being as meaningful as the reduction in repair time at that stage is measured in fractions of a second.
